web-console/knikubevirt/Create-vm/step-5-review/wizard-review.md

```diff
@@ -12,25 +12,48 @@ This encourages the user to check their settings quickly before creating the VM.
 
 ![user checks settings before creating vm](img/check-settings.png)
 
-When the user clicks “Create virtual machine” they will then be taken to the list view where they can see the VM being started and the state of the VM as "Starting".
+# States
+
+## creating State
+
+When the user clicks “Create virtual machine” they will then be will see the "creating state" for the virtual machine. In this step they would be removed from the navigation of the wizard and only able to get back in an error state where something needs to be addressed.
+
+![vm is being created](img/vm-creating.png)
+
+Alternatively, The user could navigate away from this page, In that case they'd see the VM in the list in a starting state. This will rely on the VM successfully being created without issue.
 
 ![vm is being started](img/vm-starting.png)
 
-If there is an error when the VM is being created,the user would see this in the list view (status column) and is allowed to go back and make changes (edit) in the VM's detais page.
+## Success States
+
+Upon success, they will see a success state with the option to either go to the details of the vm or go to the list view to see all vms. 
+
+![vm created](img/vm-created.png)
+
+If they are looking at the list view they will see a status of `running` or `off` if the creation was successful. 
 
-We need to distinguish between the "Try again" action that will resubmit whatever wasn’t successfully submitted (e.g. if start VM fails due to some scheduling constraints), and
-the other action suggested “Edit VM’s setup” that will take the user to the Vm’s details page where they can change settings to fix whatever is needed.
+## Error States
 
-UC: If the creation of the VM fails on the backend (e.g. some validation the frontend did not catch), it caused an error state and we want to enable the user to fix it. In this uc, the VM will not be shown in a row of the table because it does not exist, so there is no need of an error message.
+Upon error, they will see and error state with guidance to go to the step that needs updating.
+
+![vm error state](img/vm-error-state-while-creating.png)
+
+In that step that contains the error they would see the error state of the input they need to address.
```

Comment:
This makes me wonder if we should open the can of worms around validating inline after the user has filled out a field vs. waiting for the "Submit" to validate. A separate issue if you think it's worth exploring?

Reply:
ya that's a good point. Just a matter of @jelkosz opinion of implementation. Definitely a better experience.

Reply:
We do actually have some in-line validations already. We are trying to catch all errors the server might catch but it is never 100%. So even if the user passes all the UI validations, we still need to handle somehow the case that after submit the server just rejects the input.

Reply:
ok great. Ya that's certainly the best approach but good to have the backup as a fail safe :)